src/org/usfirst/frc/team467/robot/Ramps.java

```diff
@@ -0,0 +1,109 @@
+package org.usfirst.frc.team467.robot;
+
+import org.apache.log4j.Logger;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ * This class uses a state machine with three states: START, RELEASED, and DEPLOYED.
+ * Each function checks that it's in the appropriate precondition before it starts,
+ * and then sets the appropriate state when it finishes.
```

Comment:
You should describe the state machine, in this case START -> RELEASED -> DEPLOYED.